.github/ISSUE_TEMPLATE/release_testnet.md

```diff
@@ -15,39 +15,40 @@ Testnet release manager: X
 - [ ] Create new git tag e.g. `006-orthosie` on `main` (tags created on any other branches will not transfer when merged in) and push to shared remote: `git tag -a <tag_name>` - **must be annotated tag, i.e. `git tag -a`** for Vergen build. This will create a `Waiting` GitHub Action for deployment.
 - [ ] You must [manually review](https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments) the `Waiting` deployment in the GitHub Action UI before the deployment will begin. Monitor the GitHub action to ensure it completes after it is approved.
+- [ ] Update the User Guide to mention the newly created git tag.
 - [ ] Delegate to the Penumbra Labs CI validator
-- [ ] Update Galileo to run against the correct tag: change [the dependencies in the Cargo.toml](https://github.com/penumbra-zone/galileo/blob/main/Cargo.toml#L11) to reference the new git tag and commit to `main`.
-- [ ] `ssh root@galileo.penumbra.zone`and bounce Galileo via the following steps:
-  - [ ] `git checkout` latest tag in `~/penumbra`
-  - [ ] `cargo run --release --bin pcli view reset` in `~/penumbra` to reset the client state for the new testnet
-  - [ ] `killall galileo` to stop the running galileo process
-  - [ ] `git pull origin main` in `~/galileo` to update the checkout
-  - [ ] `screen -r` to start a screen session
-  - [ ] start Galileo again: `RUST_LOG=galileo=info DISCORD_TOKEN={token} cargo run --release -- serve 100penumbra 10pizza 10gm 10gn --catch-up {URL of latest unserved request}`
-  - [ ] exit `screen` (`^A d`) without stopping `galileo`
-  - [ ] **confirm that Galileo is dispensing tokens** by testing the faucet channel with your own address
-  - [ ] resupply Galileo wallet as needed
-- [ ] Make GitHub release object and draft an announcement for peer review to ensure major changes included are comprehensive.
+- [ ] Update and redeploy Galileo:
+  - [ ] Change it to run against the correct tag: change [the dependencies in the Cargo.toml](https://github.com/penumbra-zone/galileo/blob/main/Cargo.toml#L11) to reference the new git tag and commit to `main`.
+  - [ ] `ssh root@galileo.penumbra.zone`and bounce Galileo via the following steps:
+    - [ ] `git checkout` latest tag in `~/penumbra`
+    - [ ] `cargo run --release --bin pcli view reset` in `~/penumbra` to reset the client state for the new testnet
+    - [ ] `killall galileo` to stop the running galileo process
+    - [ ] `git pull origin main` in `~/galileo` to update the checkout
+    - [ ] `screen -r` to start a screen session
+    - [ ] start Galileo again: `RUST_LOG=galileo=info DISCORD_TOKEN={token} cargo run --release -- serve 100penumbra 10pizza 10gm 10gn --catch-up {URL of latest unserved request}`
+    - [ ] exit `screen` (`^A d`) without stopping `galileo`
+    - [ ] **confirm that Galileo is dispensing tokens** by testing the faucet channel with your own address
+    - [ ] resupply Galileo wallet as needed
```

Comment:
It'd be great to hook this into the future k8s deployment instead, so that this all happens automatically.

Reply:
Yes, absolutely agree: we should need to touch Galileo, we can automate the updates to it. I plan to focus on getting testnet and testnet-preview to use the new k8s setup as top priority, continuing to fuss with Galileo manually in the meantime. Once we've got the container setups running the weekly testnets, automating the Galileo changes is next up. How do those priorities sound?